.circleci/config.yml

```diff
@@ -0,0 +1,18 @@
+version: 2.1
+jobs:
+  release:
+    docker:
+      - image: cimg/node:8.17.0
+    steps:
+      - checkout
+      - run: npm install
+      - run: npm run test
+      - run: npm run semantic-release
+
+workflows:
+  release:
+    jobs:
+      - release:
+          filters:
+            branches:
+              only: master
```

Comment:
why is it only on master?

Reply:
removed, since semantic will only run on master